### ISSUE TITLE
tip numbers

### ISSUE THREAD

**Issue by xiaoyezao** [rbouckaert/DensiTree#9]
Can DensiTree deal with tree sets that have different numbers of tips in individual trees?

**Comment by rbouckaert** [comment]
Yes, DensiTree can deal with sets with different numbers of tips, **provided** the first tree contains all tips.

**Comment by RhettRautsaw** [comment]
Does this mean that if I load in a species tree and then a few thousand gene trees each with missing tips/taxa, that DensiTree should be able to handle that? I'm trying to do this, but it doesn't seem to work so I feel like I'm misunderstanding what this issue meant.

**Comment by rbouckaert** [comment]
Maybe you need to set burn-in to zero in order to pick up the first tree in the set? For newick tree files, the first tree *after burn-in* in the set must contain all tip names of the set of all the gene tree tips. Hope this makes sense.

**Comment by AlesBucek** [comment]
> Yes, DensiTree can deal with sets with different numbers of tips, **provided** the first tree contains all tips.

Hi!
This seems to be in conflict with the issue "Loading file with missing taxa needs improvement" where you described that loading trees which are missing some tips results in error: "When loading a set of trees where not all taxa are present in all trees, DensiTree cannot handle the data".
